### ISSUE TITLE
Unhandled service problems inconsistent display on tactical vs menu

### ISSUE THREAD

**Issue by roelvanmeer** [sni/Thruk#1198]
**Describe the bug**
A service problem on a host that is down is listed under the Unhandled Services link in the menu, but not as an unhandled service problem in the tactical overview. In the tactical view, it is only listed under the "on problem hosts" item.

**Thruk Version**
2.48.2-1 with livestatus 1.3.1-1

**To Reproduce**
Ensure there is a host that is down, and a service on that host that is critical. Don't ack or schedule downtime for either host or service. This service is now listed when clicking on Services (Unhandled) in the left menu, but on the tactical view there is no box with "1 Unhandled problems"

**Expected behavior**
I would expect that the number of unhandled services shown on the Unhandled Services page is identical to the total number of unhandled services as shown on the tactical overview.

**Comment by sni** [comment]
yes, that makes sense. I update the host filter to only show unhandled service issues on up/pending hosts.
